linux/just_files/just_docker_functions.bsh

```diff
@@ -517,6 +528,39 @@ function docker_defaultify()
       extra_args=${#}
       ;;
 
+    docker-compose_debug) # Run docker buildx debug build against a \
+        # docker-compose service. $1 - service name (required) $2 - docker context, defaults to "."
+      # These are not a perfect parse? Won't handle yaml strings that aren't the same syntax as bash strings
+      local compose_config=$(docker-compose config | yarp)
+      # This shouldn't be blank, as if it's absent in the compose file, docker-compose will report "Dockerfile" still
+      local dockerfile=$(sed -nE 's|^services\.'"$1"'\.build\.dockerfile = "?(.*[^"])"?|\1|p' <<< "${compose_config}")
+      if [ -z "${dockerfile}" ]; then
+        echo "Dockerfile is empty, either you forgot to or specified the wrong service name" >&2
```

Comment:
Is this commenting incomplete - `...either you forgot to` ... something?

Reply:
```suggestion
        echo "Dockerfile is empty, either you forgot to specify a service name or specified the wrong service" >&2
```